lua/org-bullets.lua

```diff
@@ -57,12 +59,22 @@ end
 ---@param line number
 ---@param conf BulletsConfig
 local function set_line_mark(lnum, line, conf)
+  local function add_symbol_padding(symbol, padding_spaces, padding_in_front)
```

Comment:
Does this need to be recreated every time this function is run, or can this function be declared outside the parent function so it is only created once and just use below?

Reply:
Good call I'll move it out